src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Public/NLHandler.cs

```diff
@@ -68,10 +68,10 @@ public class NL2FxParameters
         public Engine Engine { get; set; }
 
         /// <summary>
-        /// Set the locale that expect the output expression to be.
+        /// Set the locale name that expect the output expression to be.
         /// For example: "vi-VN" or "fr-FR".
         /// </summary>
-        public CultureInfo ExpressionLocale { get; set; }
+        public string ExpressionLocaleName { get; set; }

```

Comment:
this change isn't needed and not something we should anyways. This object Nl2FxParameters is never serialized. This object is converted to Nl2FxPayload and that is what we serialize. Can't we just replace the CultureInfo in Nl2FxPayload with the name and keep this untouched and avoiding compiler breaking change (type changes)? on the nl library side, just read the name from Nl2FxParameters.ExpressionLocale?.name and set it to the Nl2FxPayload.ExpressionLocaleName